includes/init.php

```diff
@@ -90,6 +90,9 @@
 require $install_dir . '/includes/definitions.inc.php';
 include $install_dir . '/config.php';
 
+# Uncomment the following line if you're using HTTPS and want secure session cookies
```

Comment:
Please use `//` instead of `#` just to be consistent.

We should use the same config variable we would use in setcookie() to also run this ini_set() which would be better placed in `includes/process_config.inc.php`